src/storage/compactor/src/lib.rs

```diff
@@ -89,6 +89,15 @@ pub struct CompactorOpts {
     #[clap(long, env = "RW_OBJECT_STORE_READ_TIMEOUT_MS", value_enum)]
     #[override_opts(path = storage.object_store_read_timeout_ms)]
     pub object_store_read_timeout_ms: Option<u64>,
+
+    pub is_shared_compactor: bool,
+    endpoint: String,
+    parallel_compact_size_mb: u32,
+    sstable_size_mb: u32,
+    block_size_kb: u32,
+    bloom_false_positive: f64,
+    state_store_url: String,
+    data_directory: String,
```

Comment:
You need to add the `#[clap(...)]` before each opts. Otherwise they won't show up in the startup opts.